client/src/views/SeasonView.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Season } from '../../../src/season/season'
+import axios from 'axios'
+import { SeasonRow } from '../components/SeasonRow'
+
+export interface SeasonViewState {
+    seasons: Season[]
+}
+
+export class SeasonView extends React.Component<{}, SeasonViewState> {
+    constructor(props: any) {
+        super(props)
+        this.state = {seasons: []}
+    }
+
+    componentDidMount() {
+        axios.get("http://localhost:8080/seasons").then(resp => {
```

Comment:
We could abstract the API calls later to a central point, to make it easier to manage the authorization tokens, and the hosts for dev and production, etc. No need to sweat it now, but a thing to think for the next steps

Reply:
Yeah, that makes sense :) I wanted to just get it to work, we can move that around later :)